main.js

```diff
@@ -494,6 +494,13 @@ app.on('activate', () => {
   }
 });
 
+// Defense in depth. We never intend to open webviews, so this prevents it completely.
+app.on('web-contents-created', (createEvent, win) => {
+  win.on('will-attach-webview', (attachEvent) => {
```

Comment:
`win` is a `WebContents` (not `BrowserWindow`), so e.g. `wc` might be a better name.